index.html

```diff
@@ -811,6 +908,37 @@ <h2>Generate a network error report</h2>
           </dl>
         </li>
 
+        <li>
+          If <var>report body</var>'s <code>server_ip</code> property is
+          non-empty, and not equal to <var>policy</var>'s <a>received IP
+            address</a>:

```

Comment:
Is the reasoning behind this section is to avoid sending reports to addresses that didn't send the original policy? (so "downgrading" the reports, in a sense)
If so, can you add a note on that?

Reply:
Exactly!  There is some text down in the privacy § describing the rationale but I added a note here, too, so that there's an explanation close to the algorithm step.